classes/classes/Creature.as

```diff
@@ -140,7 +140,16 @@ package classes
 				
 		//Appearance Variables
 		//Gender 1M, 2F, 3H
-		public var gender:int = GENDER_NONE;
+		public function get gender():int
+		{
+			if (hasCock() && hasVagina())
```

Comment:
I would recommend using `{ }` for the if statements.
```
if (foo == 1) {
    doThis();
}
```
instead of
```
if (foo == 1)
    doThis();
```

Although it will require more lines, this will reduce the chances for an error when someone edits the code.